src/main/java/Task.java

```diff
@@ -0,0 +1,55 @@
+public class Task {
+
+    protected String description;
+    protected String eventDate;
+    protected boolean isDone;
+
+    enum Level {
+        low,
+        average,
+        high
+    }
+
+
+    /**
+     * Main constructor that accepts a description of the task, and by default
+     * assigns a false value to the undone task.
+     * @param description Description of task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.eventDate = "";
+    }
+
+    /**
+     * Overriden method to mark a task as done.
```

Comment:
Did you mean "Overloaded" here as well? I've noticed it in a few other places. Otherwise, as the other reviewer mentioned, overridden methods should tagged with "@Override"